dev/io.openliberty.microprofile.config.2.0.internal_fat/fat/src/io/openliberty/microprofile/config/internal_fat/Config20ExceptionTests.java

```diff
@@ -77,42 +82,57 @@ public void testBadObserver() throws Exception {
         assertTrue("error not found: " + errors.size(), errors.size() > 0);
     }
 
+    // Check an appropriate error message occurs when a user tries to Inject a non-existing Config Property without a "name" field into a Method.
     @Test
     public void testMethodUnnamed() throws Exception {
+        String beanDir = ConfigUnnamedMethodInjectionBean.class.getName();
         List<String> errors = server
-                        .findStringsInLogs("SRCFG02002: Could not find default name for .*io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnamedMethodInjectionBean.*setSimpleKey6");
-        assertNotNull("error not found", errors);
-        assertTrue("error not found: " + errors.size(), errors.size() > 0);
+                        .findStringsInLogs("SRCFG02002: Could not find default name for @ConfigProperty InjectionPoint \\[BackedAnnotatedParameter\\] Parameter 1 of \\[BackedAnnotatedMethod\\] @Inject public "
+                                           + beanDir + ".aMethod\\(@ConfigProperty String\\)");
```

Comment:
It's usually better to be less specific than this when checking error messages because it makes the test brittle if the error message, or the `toString()` method on one of the involved classes changes.

A search regex like this might be better: `"SRCFG02002: .*" + beanDir + ".aMethod"`, ideally with a comment showing an example of the type of log line it's looking for.